test/integration/download-known-assets.js

```diff
@@ -65,9 +66,9 @@ const testAssets = [
     }
 ];
 
-test('addWebSource', t => {
+test('addWebStore', t => {
```

Comment:
`addWebSource` is deprecated and has been renamed to `addWebStore`, so this change eliminates a warning